src/webui/webapplication.cpp

```diff
@@ -179,7 +179,7 @@ WebApplication::WebApplication(IApplication *app, QObject *parent)
             LogMsg(tr("Unacceptable session cookie name is specified: '%1'. Default one is used.")
                    .arg(m_sessionCookieName), Log::WARNING);
         }
-        m_sessionCookieName = DEFAULT_SESSION_COOKIE_NAME;
+        m_sessionCookieName = DEFAULT_SESSION_COOKIE_NAME + QString::number(m_webUiPort);
```

Comment:
An alternative would be to come up with a unique cookie name on startup.

Reply:
IMO, the current method looks better, since it still identifies the cookie name with a specific qBittorrent instance, so that some client with a smarter implementation than browsers (which is why the problem is) could even send it to a specific instance and not to everyone on the same host.